await-generator/src/SOFe/AwaitGenerator/Await.php

```diff
@@ -52,6 +52,9 @@ class Await extends PromiseState{
 	/** @internal Use `Await::race` instead */
 	public const RACE = "race";
 
+	/** @var bool */
+	private static $warnedDeprecatedDirectYield = false;
+
```

Comment:
```suggestion

	public static function disableDeprecatedDirectYieldWarning(): void {
		self::$warnedDeprecatedDirectYield = true;
	}
```

Reply:
@keopiwauyu tell plugin coder to use `yield from $generator` instead of `yield $generator`, instead of disabling this warning.

Reply:
@keopiwauyu this is not even a confusing message. It tells them exactly what they should do to fix this.

Reply:
Use of the old syntax will cause issues in the future, such as inability to use #106.

Reply:
> @keopiwauyu this is not even a confusing message. It tells them exactly what they should do to fix this.

it confuse to server owner not plugin coder of course plugin owner know what err this is

but server owner will go pmmp discord spam screenshot INTERNAL SERVER ERROR and i dont like thdi

Reply:
@keopiwauyu there is no internal server error. it is just an error echo, once.

virions are not auto-updated by the user, it always comes with the plugin build, so the developer can notice it in advance.